rail_scripts/rail-preprocess-parquet

```diff
@@ -181,14 +187,15 @@ def preprocess(cfg):
                 print(input_)
                 return
 
-            has_magnitude, has_flux, has_ra, has_dec = parquet_check_column_presence(
+            has_magnitude, has_flux, has_ra, \
+            has_dec, has_objectId = parquet_check_column_presence(
                     f, flux_columns + flux_error_columns,
                     mag_columns + mag_error_columns, 
-                    ra_column, dec_column)
+                    ra_column, dec_column, objectId_column)
```

Comment:
This new variable does not exist. Executing this line gives the error:
```
Traceback (most recent call last):
  File "/lustre/t1/cl/lsst/tmp/henrique.almeida/slurm-home/bin/rail-preprocess-parquet", line 238, in <module>
    if __name__ == '__main__': main()
                               ^^^^^^
  File "/lustre/t1/cl/lsst/tmp/henrique.almeida/slurm-home/bin/rail-preprocess-parquet", line 236, in main
    preprocess(cfg)
  File "/lustre/t1/cl/lsst/tmp/henrique.almeida/slurm-home/bin/rail-preprocess-parquet", line 195, in preprocess
    ra_column, dec_column, objectId_column)
                           ^^^^^^^^^^^^^^^
NameError: name 'objectId_column' is not defined
```

A simple way to fix it is to define it like the other extra columns:
```
    objectId_column = cfg.objectId
```

Maybe all the extra columns should be bundled in a list for easier usage.

Reply:
Inconsistent coding style: please use underscore notation for variable names: `object_id_column`.